### PR TITLE
Feature/50 location

### PR DESCRIPTION
**Pull request by wenwenchenbosch**

### CODE REVIEW

File: invehicle-apps/kuksa-traccar-client/providers/kuksa/clientComm.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/python3
```
Comment (wenwenchenbosch) [inline invehicle-apps/kuksa-traccar-client/providers/kuksa/clientComm.py]: @SebastianSchildt This file is duplicated to https://github.com/eclipse/kuksa.val/blob/master/vss-testclient/clientComm.py. What should we do?

Comment (wenwenchenbosch) [reply invehicle-apps/kuksa-traccar-client/providers/kuksa/clientComm.py]: create issue https://github.com/eclipse/kuksa.val/issues/111 to solve the problem

**Comment by wenwenchenbosch** [comment]
@nayakned ready to review again

**Comment by nayakned** [comment]
Tested this PR along with the "cleanup" branch of the kuksa.val repository.
The changes can be merged.

We should perhaps think about an end-to-end (from kuksa.val server to traccar ecosystem and gpsd etc.) documentation to deploy this application in a tutorial style.

**Comment by wenwenchenbosch** [comment]
@SebastianSchildt: Naresh @nayakned approved the PR, and you?

**Review by SebastianSchildt — APPROVED** [review]

**Pull request merged**